_posts/2019-09-30-team-working-agreements.md

```diff
@@ -0,0 +1,224 @@
+      spending many sprints trying to guess.
+    </p>
+    <p>
+      This might be short-sighted, and maybe my opinion will change over time, but I have seen the most value out of
+      *the actual brainstorm* than the agreement itself. My teams have learned so much about each others' habits and
+      values.
+    </p>
+  </div>
+  <div class="answer">
+    <img src="/images/artsy-engineering-hiring/adam.jpg">
+    <p class="intro">Adam Iskounen says...</p>
+    <p>
+      TODO Adam: put this in your own words/expand/do what you want here
+      The process has helped confirm that the team shared interest in team dynamics & the process of collaboration 
+      across the team. As a new member it's nice to know that information.
+    </p>
+  </div>
+</aside>
+
+### It can lead to better engagement in sprint activities.
+
+One of our teams established in their agreement that sprint meetings were valuable. This might seem too obvious to
+put in an agreement, but it portrays clearly to everyone that they should attend all sprint meetings and be
+actively engaged.
+
+The same team got a lot out of making slogans for the items in their agreement. Phrases like "Pair by default" and
+"Incremental improvement over consistency" were introduced during the brainstorm. They've since become mantras for
+the team to refer to during sprint work and ceremonies. Team values and habits solidify through this kind of
+repetition.
+
+### The agreement provides on-boarding documentation.
+
+When new members join a team, it takes time for them to adjust. A team working agreement provides an up-to-date
+reference on the team's preferred methods of working. The agreement isn't set in stone, and as the team changes
+it's important to update it...but having it in writing helps new members acclimate quickly.
+
+## What are some challenges with Team Working Agreements?
+
+### Uncertainty about what the agreement should include.
+
+Some teams have struggled to identify what belongs on a team agreement. Should it describe our team values? Habits?
+Procedures? Rules?
+
+It likely varies from team to team what you want here. It's definitely important that you identify what you're
+looking for up front, and make it clear heading into the brainstorm. Suggest categories, sample habits, & questions
+to ask heading into the brainstorm, and keep them visible during the meeting.
+
+Be cautious about introducing individual bias, though.
+
+<aside>
+  <div class="answer">
+    <img src="/images/artsy-engineering-hiring/steve.jpg">
+    <p class="intro">Steve Hicks says...</p>
+    <p>
+      Having been through a brainstorm on another team where I felt like we spent too much time talking about things I deemed "too specific," I made a strong effort on another team to constrain the suggestions. To me, team values were really important for this new team - and I did such a good job of encouraging those types of suggestions that it's pretty much all we ended up with.
```

Comment:
> constrain the suggestions

It might help clarify what you mean here if you could include a concrete example?

Reply:
👍 I'll try to come up with one.

Reply:
I couldn't remember a specific example, but made the wording a little more specific.